relayer/config/config.go

```diff
@@ -65,8 +66,11 @@ type Config struct {
 	SignatureCacheSize     uint64                   `mapstructure:"signature-cache-size" json:"signature-cache-size"`
 	ManuallyTrackedPeers   []*basecfg.PeerConfig    `mapstructure:"manually-tracked-peers" json:"manually-tracked-peers"`
 	AllowPrivateIPs        bool                     `mapstructure:"allow-private-ips" json:"allow-private-ips"`
+	TLSCertPath            string                   `json:"tls-cert-path,omitempty"`
+	TLSKeyPath             string                   `json:"tls-key-path,omitempty"`
 
 	// convenience field to fetch a blockchain's subnet ID
+	tlsCert                *tls.Certificate `json:"-"`
```

Comment:
Does `json:"-"` do something special?

Reply:
Makes it not serialize but also unnecessary since it's not exported. Thanks